### DISCUSSION TITLE
Add Style file as a Link

### DISCUSSION THREAD

**Discussion by CADdentRene** [AvaloniaUI/Avalonia#8904] [General]
I have a Repository where i have defined different Avalonia (Control) Styles.

Now if i want to use them in a Project i have to add them to my current Project as a LINK, so the original source will be overwritten once i edit it.

But i cant use "StyleInclude" to use the linked Styles... It always throws the following error: "Avalonia.Markup.Xaml.XamlLoadException: "No precompiled XAML found for avares://WebShop/Assets/CADSvgImages.axaml (baseUri: avares://WebShop/App.axaml), make sure to specify x:Class and include your XAML file as AvaloniaResource"

Any solutions?

![image](https://user-images.githubusercontent.com/102605287/188894085-0e1e374e-daed-4a63-af14-5549357d0afb.png)

![image](https://user-images.githubusercontent.com/102605287/188894039-7991c910-2b4c-4f96-a73c-100a295fbd10.png)

**Comment by maxkatz6** [comment]
Link files are not supported
